components/director/internal/domain/tenant/converter.go

```diff
@@ -21,8 +23,13 @@ func (c *converter) ToEntity(in *model.BusinessTenantMapping) *tenant.Entity {
 		ID:             in.ID,
 		Name:           in.Name,
 		ExternalTenant: in.ExternalTenant,
-		ProviderName:   in.Provider,
-		Status:         tenant.TenantStatus(in.Status),
+		Parent: sql.NullString{
+			String: in.Parent,
+			Valid:  true,
```

Comment:
should be set to true only if in.Parent in not empty string or nil

Probably the best would be to set defaults for BusinessTenantMapping.Parent to be empty string - "", and later here just set this to len(in.Parent)!=0

Reply:
Good catch! But I was planning to use [this function](https://github.com/kyma-incubator/compass/pull/1888/files#diff-e4867418df28596ed6cf910ad54f304cc1b560eb25e35a1c327297f6ed90f657R40) instead since it resolves exactly this issues. 😄